### DISCUSSION TITLE
Confusion about some basic concepts

### DISCUSSION THREAD

**Discussion by Ethan-xj** [idaholab/moose#21583] [Q&A General] [answered]
Hi all, I'm just starting to use moose.
I want to know the definition and relationship of node, element and block.

Could anyone kindly teach me?

Best regards,
Ethan

**Comment by GiudGiud** [comment, marked as answer]
Hello

This page might help
https://mooseframework.inl.gov/source/mesh/MooseMesh.html

in general any good book about finite-elements will be really helpful for these concepts

Guillaume

**Reply by Ethan-xj** [reply]
Thank you for your reply!
